test/correct-filename.test.js

```diff
@@ -1,73 +1,70 @@
-var should = require("should");
-var path = require("path");
-var fileLoader = require("../");
+const fileLoader = require('../');
```

Comment:
Also needs to be moved into the `src` folder. Tests should reside with the code they are testing ( or that is what was discussed ).

The `build`  task ignores them though we need to put thought into how we want to handle libs with extensive `fixtures`